modules/ROOT/partials/integrations/svelte-quick-start.adoc

```diff
@@ -39,57 +44,73 @@ endif::[]
 ifeval::["{productSource}" == "cloud"]
 
 . Open `+src/App.svelte+` and add:
-* An `+import+` statement for the {productname} component.
+* An `+import+` statement for the {productname} Svelte component.
 * The `+<Editor />+` as a placeholder for the editor.
 +
 For example:
 +
 _File:_ `+src/App.svelte+`
 +
-[source,html]
+[source,html,subs="+macros"]
 ----
-<script lang="ts">
+<script>
 import Editor from '@tinymce/tinymce-svelte';
+let conf = {
+  height: 500,
+  menubar: false,
```

Comment:
The Vite template has this behaviour where the `margin: 0 auto;` styling in `app.css` will cause the TinyMCE editor to move around as the user opens up the menubar because the App component size is changing. The easiest workaround I could find for this was to disable the menubar which is what TinyMCE React does to work around this problem. There might be a better solution to this.

Reply:
Have we created a ticket to investigate this issue further?

Reply:
Investigation ticket created: [QA-1211](https://ephocks.atlassian.net/browse/QA-1211)